README.md

```diff
@@ -1,10 +1,10 @@
-# The Open Defense Fund for Cultural Humanity
+# The Open Defense Fund for the Decentralized Web
 
-Owned by none, ODF is a consortium of organizations participating in an open forum dedicated to protecting the future of the web. No member speaks for or represents the foundation or any other member in the think tank, all views come from a diverse range of ethnic and cultural heritages.
+Owned by none, ODF is a consortium of organizations participating in an open forum dedicated to protecting the future of the web. No member speaks for or represents the foundation or any other member in the think tank.
 
 ### Mission
 
- > ODF's mission is to facilitate healthy discourse between differing cultures by freely promoting corroborative facts with fair algorithms.
+ > ODF's mission is to facilitate open source projects that aim to further protect the web from monopoly, whether cultural or technological.
```

Comment:
The web is already ...90% monopolized and pretty much a monoculture because of it (white consumerism, is that a culture?)

Reply:
"further" word should be removed?